plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/handlers/SQLEditorHandlerToggleTabPinned.java

```diff
@@ -0,0 +1,48 @@
+package org.jkiss.dbeaver.ui.editors.sql.handlers;
+
+import java.util.Map;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.ui.IEditorPart;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.commands.IElementUpdater;
+import org.eclipse.ui.handlers.HandlerUtil;
+import org.eclipse.ui.menus.UIElement;
+import org.jkiss.dbeaver.ui.editors.sql.SQLEditor;
+import org.jkiss.dbeaver.ui.editors.sql.internal.SQLEditorMessages;
+import org.jkiss.dbeaver.utils.RuntimeUtils;
+
+public class SQLEditorHandlerToggleTabPinned extends AbstractHandler implements IElementUpdater {
+
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException {
+        SQLEditor editor = RuntimeUtils.getObjectAdapter(HandlerUtil.getActiveEditor(event), SQLEditor.class);
```

Comment:
Please correct indentation. We use 4 spaces.

Reply:
Fixed in [98eae4](https://github.com/hawthorne3341/dbeaver/commit/998eae41b2276f66eb82bf18e88a135bbfb88116)

Reply:
There are tabs instead of 4 spaces now

Reply:
Have attempted to correct in [2e1837](https://github.com/hawthorne3341/dbeaver/commit/2e1837e76930e106c8358390c97c967b681529b1)

There are still some style checks failing: missing javadoc comment, order of import statements, and `if construct must use []s`. Are these enforced? If so, I will correct, but can you clarify what `if construct must use []s` means? Was not able to find information at:
https://google.github.io/styleguide/javaguide.html